modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -158,6 +158,37 @@ module UpdatePoaRequest
         end
       end
 
+      ##
+      # PersonWebServiceBean
+      #
+      module PersonWebServiceBean
+        DEFINITION =
+          Bean.new(
+            path: 'PersonWebServiceBean',
+            namespaces: Namespaces.new(
+              target: 'http://person.services.vetsnet.vba.va.gov/',
+              data: 'http://person.services.vetsnet.vba.va.gov/'
```

Comment:
do we need the duplication? seems weird to have two different items here.. unless it's to prepend the `/data` part?

Reply:
Yeah if there isn't a distinct second namespace that gets used inside the SOAP body for this bean, then the one parent namespace will already apply to all elements.